task/flake8/0.1/README.md

```diff
@@ -0,0 +1,50 @@
+# flake8
+
+The task provides linting based on [flake8](https://pypi.org/project/flake8/) for Python.
+This is a blatant copy of the [pylint](https://github.com/tektoncd/catalog/tree/main/task/pylint/0.2) task.
+
+## Install the Task
```

Comment:
can this be removed :slightly_smiling_face: ?

Reply:
Done